bin/sign

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+ENV["PROJECT_ROOT"] = Dir.pwd
+
+# Begin warning: This sign script is an internal tool.
+# This is not intended to function outside of Puppet Labs' infrastructure.
+# End of warning.
+
+if Dir["output/**/*"].select { |entry| File.file?(entry) }.empty?
+  fail "No packages to sign in the output directory. Maybe you want to build some first?"
+else
+  require 'packaging'
+  Pkg::Util::RakeUtils.load_packaging_tasks
+  Pkg::Util::RakeUtils.invoke_task('pl:jenkins:sign_all', 'output')
+end
```

Comment:
nitpick, but you could take the packaging stuff out of the `else` and just call it after the failure case.